### PR TITLE
Add Alienware Icon

### LINKED ISSUE #9319: Request icon AlienWare
### Brand Name

Alienware

### Website

https://www.dell.com/es-es/gaming/alienware

### Popularity Metric

Global Rank #987

### Official Resources for Icon and Color

None

### Additional Comments

La Marca es la gama alta de PCs para gaming de Dell

### Contributing

Yes

### PR DESCRIPTION
**Pull request by RiteshK-611**
**Issue:** closes #9319

**Similarweb rank:**
Global Rank: 987

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
This commit adds Alienware Icon 

### Preview
alienware

### CODE REVIEW

**Review by FraneCaleta — APPROVED** [review]

File: _data/simple-icons.json [hunk 1]
```diff
@@ -520,6 +520,11 @@
             "hex": "FF6A00",
             "source": "https://www.alibabagroup.com/en/ir/reports"
         },
+        {
+            "title": "Alienware",
+            "hex": "000000",
+            "source": "https://www.dell.com/es-es/gaming/alienware"
```
Comment (adamrusted) [inline _data/simple-icons.json:526]:
```suggestion
            "source": "https://www.dell.com/en-us/gaming/alienware"
```

If we can't go for a language-less website, probably safest to go with an english one.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -520,6 +520,11 @@
             "hex": "FF6A00",
             "source": "https://www.alibabagroup.com/en/ir/reports"
         },
+        {
+            "title": "Alienware",
+            "hex": "000000",
```
Comment (adamrusted) [inline _data/simple-icons.json:525]:
```suggestion
            "hex": "541BAE",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Icon is spot on, thank you @RiteshK-611!
Couple of minor changes to the JSON that I'd make, one being the mid-point of the purple gradient used frequently throughout the site, and the other converting the source URL to an english variation.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @RiteshK-611 🎉

**Pull request merged**